README.md

```diff
@@ -86,6 +86,19 @@ To run the script in powershell, from the repository root, run:
 ### Generating Responsible AI Dashboard insights
 Once you have created an Azure workspace and registered your components in the one-time setup above, you can create a Responsible AI dashboard via the CLI or SDK. Start here for `examples` [folder](examples) to get started.
 
+#### Limitations
+
+The current set of components have a number of limitations on their use:
+
+- All models must be in registered in AzureML in MLFlow format with an sklearn flavour
+- The models must be loadable in the component environment (largely based on the `responsibleai` package)
```

Comment:
Do we need to say it is based on the `responsibleai` package? If it's not that important, I would drop this part.